examples/hello_world/chrome/build_fetch.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -ex
+
+# This is the same as the directory above this.
```

Comment:
Could the comments here be tweaked slightly to document what `--fetch` is and what the difference is from `./build.sh`?